lib/scholar/metrics/distance.ex

```diff
@@ -13,6 +13,11 @@ defmodule Scholar.Metrics.Distance do
   D(x, y) = \\sqrt{\\sum_i (x_i - y_i)^2}
   $$
 
+  ## Options
+
+    * `:axes` - Axes to aggregate distance over. If `:axes` set to `nil` then function does not aggregate distances.
```

Comment:
What does it mean to not aggregate distances?

Reply:
Perhaps an example with multi-dimensional inputs and `axes: nil` could be useful here